src/EnergyPlus/UnitarySystem.cc

```diff
@@ -474,7 +474,7 @@ namespace UnitarySystems {
             }
         }
 
```

Comment:
note as reminder that UnitarySystem can't set fan volume flow until components have sized. this is a defect that needs to be corrected.